src/renderer/components/NodeDocumentation/NodeExample.tsx

```diff
@@ -98,6 +98,14 @@ export const NodeExample = memo(({ selectedSchema }: NodeExampleProps) => {
         [setOutputHeight]
     );
 
+    const [nickname, setNickname] = useState<string | undefined>(undefined);
+    const setSingleNickname = useCallback(
+        (value: string | undefined): void => {
+            setNickname(value);
+        },
+        [setNickname]
+    );
```

Comment:
Why not just use `setNickname` directly?

Reply:
because i blindly copied this and didnt think about what it was actually doing lol